doc/sphinx/language/extensions/canonical.rst

```diff
@@ -544,12 +546,10 @@ We need some infrastructure for that.
     Notation "'Error : t : s" := (unify _ t (Some s))
       (at level 50, format "''Error' : t : s").
 
-    Open Scope string_scope.
-
   End infrastructure.
 
 To explain the notation ``[find v | t1 ~ t2]`` let us pick one of its
-instances: ``[find e | EQ.obj e ~ T | "is not an EQ.type" ]``. It should be
+instances: ``[find e | EQ.obj e ~ T | Is_not_an_EQ_type ]``. It should be
```

Comment:
How is this related to splitting off stdlib??  Seems like it should be in a separate PR.